TRIP-21635 update deep links to shopping results

FriederikeKr opened the pull request:
Updating the existing deep link to shopping results documentation for the upcoming June release. We will release deep links to car, hotel and rail shopping results and update the path of the link for air.

RoAnnC1 commented:
Hi @FriederikeKr The email you used to create the commit  is not associated with your GitHub account. Make sure to add (and verify) your email to your GitHub account.

Afterwards you can trigger a recheck via https://cla-assistant.io/check/SAP-docs/preview.developer.concur.com?pullRequest=1081

FriederikeKr commented:
thanks. for my first commit, my local git configuration was incorrect. I changed that and rebased / force-pushed without any effect. I will try again on a new branch with new commits: https://github.com/SAP-docs/preview.developer.concur.com/pull/1086
